pkg/storage/etcd/etcd_helper_test.go

```diff
@@ -123,7 +121,7 @@ func TestList(t *testing.T) {
 	var got api.PodList
 	// TODO: a sorted filter function could be applied such implied
 	// ordering on the returned list doesn't matter.
-	err := helper.List(context.TODO(), key, "", storage.Everything, &got)
+	err := helper.List(context.TODO(), "/", "", storage.Everything, &got)
```

Comment:
So let me answer all questions here.

We have created etcd helper with etcdtest.PathPrefix():
```
helper := newEtcdHelper(server.Client, testapi.Default.Codec(), etcdtest.PathPrefix())
```

Later it creates objects using the object name, e.g. "foo", "bar" as the key:
```
err := createObj(t, helper, list.Items[i].Name, &list.Items[i], returnedObj, 0)
```

When we call List(), if we use etcdtest.PathPrefix() again, it will be "etcdtest.PathPrefix()/etcdtest.PathPrefix()". We should list "/" instead, which will be prefixed inside helper.